### ISSUE TITLE
Create `front-token` and send it whenever sending access token to the frontend

### ISSUE THREAD

**Issue by rishabhpoddar** [supertokens/supertokens-go#3]
The `front-token` allows you to read user id and jwt payload securely from the frontend without exposing the access token

- In this [file](https://github.com/supertokens/supertokens-go/blob/master/supertokens/cookieAndHeaders.go) create the `attachFrontTokenInHeaders` function similar to the one defined [here](https://github.com/supertokens/supertokens-node/blob/master/lib/ts/cookieAndHeaders.ts#L215).
- The `attachFrontTokenInHeaders` function should set the `frontTokenHeaderKey` and `tokenInfo` in the header. The `front-token` should be exposed by setting `front-token` as the value of the `Access-Control-Expose-Headers` key in the header.
- Make sure that whenever the access token is attached to the response the `front-token` is also attached, this will be done by calling the `attachFrontTokenInHeaders` function.

Run all the tests and make sure that they all pass.

**Comment by vidu171** [comment]
Hi, I think I understand what is to be done here.
Can I take this issue?

**Comment by rishabhpoddar** [comment]
Sure @vidu171. I have assigned this issue to you.

**Comment by rishabhpoddar** [comment]
@vidu171 when creating a PR, please put the target branch as `1.4`. Thanks

**Comment by vidu171** [comment]
@rishabhpoddar thanks, I will make sure of doing that

**Comment by vidu171** [comment]
Hi,
I have made the required changes.
All the test cases except one passed
the test `TestAppendingToExistingHeader` failed.
After doing some digging I found that the **test case is incomplete**.

The test case is expecting `customValue, id-refresh-token, anti-csrf` as the `Access-Control-Expose-Headers` values.
Whereas I am sending `customValue, front-token, id-refresh-token, anti-csrf`

```
if res.Header.Get("Access-Control-Expose-Headers") != "customValue, id-refresh-token, anti-csrf" {
		t.Error("header value not set")
	}
```
After modifying the test case to
```
if res.Header.Get("Access-Control-Expose-Headers") != "customValue, front-token, id-refresh-token, anti-csrf" {
		t.Error("header value not set")
	}
```
All the test cases pass. 
Should I include my test case changes with my PR?
